### ISSUE TITLE
Deprecated support for desktop apps

### ISSUE THREAD

**Issue by DRSDavidSoft** [michaelsboost/WebDGap#10]
In your README file, you've specified that native desktop apps are deprecated, and it seems that you've instead focused on Chrome apps.

However, it seems that Google has declared the Chrome apps itself deprecated: 
https://blog.chromium.org/2016/08/from-chrome-apps-to-web.html

Could you elaborate on the future of WebDGap?

**Comment by michaelsboost** [comment]
Because Google discontinued Chrome packaged applications that feature will be removed off of WebDGap. However, popup extensions and desktop exportation will still remain applicable. 

The future of WebDGap relies on mobile support. It's easy for desktop users to make mobile applications and mobile users to make mobile apps now but not visa versa. (and for good reason) I plan on making it available for iOS, however, time and funding are a big factor as to why that hasn't become a reality yet. 

For desktops, a lot of times (but not always) when a new operating system is released the wrapper needs to be updated to meet new requirements. The plan with WebDGap is to have an all in one wrapper that saves you time so you don't need to compile for each platform individually. 

Does that answer your question?

**Comment by DRSDavidSoft** [comment]
Yes, that answers my question, and thanks for the detailed explanation.

Would you be able to also invest on WebDGap as an electron module to be able to use the shared development on the electron platform?

**Comment by michaelsboost** [comment]
As of right now in my life no. I cannot spend the time to research it in order to make that happen. 

It is open source though so if anyone wants to feel free just drop the link down and let me know so I can update the documentation.
